modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -0,0 +1,152 @@
+[id="cluster-logging-exported-fields-top-level-fields_{context}"]
+== Top-level fields
+
+// Normally, the preceding title would be an H1 prefixed with an `=`. However, because the following content is auto-generated at https://github.com/ViaQ/documentation/blob/main/src/data_model/public/top-level.part.adoc and pasted here, it is more efficient to use it as-is with no modifications. Therefore, to "realign" the content, I am going to prefix the title with `==` and use `include::modules/cluster-logging-exported-fields-top-level-fields.adoc[leveloffset=-1]` in the assembly file.
+
+// DO NOT MODIFY THE FOLLOWING CONTENT. Instead, update https://github.com/ViaQ/documentation/blob/main/src/data_model/model/top-level.yaml and run `make` as instructed here: https://github.com/ViaQ/documentation
+
+The top level fields may be present in every record.
+
+
+
+=== message
+
+Original log entry text, UTF-8 encoded
+
+[horizontal]
+Data type:: text
+
+Example value:: `HAPPY`
+
+
+
+
+
+=== @timestamp
+
+A UTC value that marks when the log payload was created or when the log payload was first collected if the creation time is not known. This value is the log processing pipeline's “best effort” determination of when the log payload was generated. The “@” prefix denotes a field that is reserved for a particular use. By default, most tools look for “@timestamp” with ElasticSearch.
+
+[horizontal]
+Data type:: date
+
+Example value:: `2015-01-24 14:06:05.071000000 Z`
+
+
+
+
+
+=== hostname
+
+The FQDN of the entity generating the original payload. This field is a best-effort attempt to derive this context. Sometimes, the entity generating it knows it; other times, that entity has a restricted namespace itself, and the collector or normalizer knows that.
```

Comment:
Same as last comment. Is a best-effort attempt an estimate? I'm not sure if the Sometimes, the entity sentence explaining estimation is necessary.

Reply:
Too many instances of "it" and "that" create null pointer errors for me! 
@alanconway, Can I simply remove "Sometimes, the entity generating it knows it; other times, that entity has a restricted namespace itself, and the collector or normalizer knows that." Or with your help, should I provide a better explanation?

Reply:
"The name of the host where this log message originated. In a kubernetes cluster this is the same as `kubernetes.host`." 
FYI: The Viaq model was originally designed to be used more generally than kubernetes, but for present purposes we just need the kubernetes case to be clear.